dts/bindings/mtd/atmel,at45.yaml

```diff
@@ -45,6 +45,18 @@ properties:
     required: true
     description: Flash page size in bytes.
 
+  workaround-broken-read:
+    type: boolean
+    description: |
+      Some AT45 flash chips do not support the regular read opcode. If set, the driver will perform
+      read with 0BH opcode [Continuous Array Read (High Frequency Mode)] instead.
+
+  workaround-broken-write:
+    type: boolean
+    description: |
+      Some AT45 flash chips do not support the regular write/modify opcodes that combined two
```

Comment:
Mention the opcodes here.
I think that both descriptions should also have mention that once the properties are set then every device will be treated as broken because there is no way to identify.
Also rather than writing "Some AT45 flash chips do not support", write something like "You may encounter AT45 devices that are expected to support command but" . Because some AT45 may indicate that there is AT45xxx that supports and AT45yyy that does not. These are different devices, and you can basically identify them by markings, and probably should be handled with different paths in driver execution.

The workaround is for something you expect to work in certain way, but it does not for some reason.

Reply:
updated the description